### PR TITLE
Added Polio Prevenntion 

### LINKED ISSUE #429: 📝[Docs]: Polio
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue Description

Polio, caused by the poliovirus, was once a feared disease causing paralysis and death worldwide. The development of the polio vaccine by Jonas Salk in 1955 and later the oral vaccine by Albert Sabin significantly reduced its incidence.

### Suggested Change

Prevention: Vaccination remains the most effective prevention method. Global efforts continue to completely eradicate polio.

### Rationale

_No response_

### Urgency

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

### PR DESCRIPTION
**Pull request by Maheshwari-Love**
#### PR Description
Please include a summary of the changes and the related issue. Describe your changes in detail.
 closed #429 
#### Type of Change
- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [x] Documentation update

#### Select your work-area
- [ ] Frontend
- [ ] Backend
- [x] Documentation
- [ ] Others

#### Related Issue
Please provide a link to the issue solved if applicable.

#### Add your Work Example
📷 Add a Snapshot

#### Fixes (mention the issue number which this fixes)
#429 

#### Checklist
- [x] I have updated my branch and synced it with the project's 'develop' branch before making this PR.
- [x] I have optimized the file changes.
- [x] I have added a snapshot of my work example.
- [x] I have made a PR to the project's develop branch.

#### Undertaking
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] I have checked for plagiarism and assure its authenticity.
- [x] I have read and followed the code of conduct for this repository. I understand that violation of this undertaking may have legal consequences.

- [x] I Agree

### CODE REVIEW

**Comment by Maheshwari-Love** [comment]
@SB2318 please review this pr

**Review by SB2318 — CHANGES_REQUESTED** [review]
Explore your topics, add more info. Include images.

**Comment by Maheshwari-Love** [comment]
> Explore your topics, add more info. Include images.

Done!!

File: Polio-Preventions.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+### Vaccination Status
+Vaccination is the most effective way to prevent polio. Individuals who have not received the full course of the polio vaccine are at a significantly higher risk of contracting the disease. The oral polio vaccine (OPV) and inactivated polio vaccine (IPV) are both highly effective in preventing polio, and widespread vaccination campaigns have been key in reducing global cases. However, incomplete vaccination or missed doses can leave individuals vulnerable.
+
+### Hygiene and Sanitation
+Poor hygiene and sanitation practices are major contributors to the spread of the poliovirus. In areas where open defecation is common or where clean water is scarce, the virus can easily contaminate food and water supplies, leading to widespread transmission. Proper handwashing with soap, safe disposal of human waste, and ensuring access to clean drinking water are critical measures to prevent the spread of polio.
+
+## Prevention Strategies
+Preventing polio involves a combination of vaccination, good hygiene practices, and community education.
+
+### Vaccination
```
Comment (SB2318) [inline Polio-Preventions.md:69]: Explore vaccination (Our primary focus will be on polio vaccination and the significance of social awareness surrounding it. We will gradually explore the potential consequences of not vaccinating children against polio, highlighting the severe health risks they may face. Following this, we will delve deeper into the effects of polio itself, discussing how affected individuals can survive and manage the disease.)

Source : https://www.cdc.gov/polio/vaccines/index.html

Now, I believe you have enough information  to restructure your content.

Comment (Maheshwari-Love) [reply Polio-Preventions.md:69]: > Explore vaccination (Our primary focus will be on polio vaccination and the significance of social awareness surrounding it. We will gradually explore the potential consequences of not vaccinating children against polio, highlighting the severe health risks they may face. Following this, we will delve deeper into the effects of polio itself, discussing how affected individuals can survive and manage the disease.)
> 
> Source : https://www.cdc.gov/polio/vaccines/index.html
> 
> Now, I believe you have enough information to restructure your content.

@SB2318 Updated... if any changes

**Review by SB2318 — APPROVED** [review]
Looks good😇 Thanks for your contribution!  🎉

**Pull request merged**